libmamba/src/specs/version.cpp

```diff
@@ -0,0 +1,562 @@
+// Copyright (c) 2023, QuantStack and Mamba Contributors
+//
+// Distributed under the terms of the BSD 3-Clause License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+
+#include <algorithm>
+#include <cassert>
+#include <charconv>
+#include <iterator>
+#include <optional>
+#include <tuple>
+
+#include "mamba/core/error_handling.hpp"
+#include "mamba/core/util_cast.hpp"
+#include "mamba/core/util_string.hpp"
+#include "mamba/specs/version.hpp"
+
+namespace mamba::specs
+{
+    namespace
+    {
+        // TODO(C++20) use operator<=>
```

Comment:
Note: most tools detecting TODOs will detect `TODO:` (with the colon) specifically.

Reply:
Hopefully they won't stay here for too long ;)